### PR TITLE
highlights(php): declare directives and `:`

### PR DESCRIPTION
**Pull request by zweimach**
Is `@preproc` the right capture for this?

| Before | After |
| ------ | ----- |
| image | image |

### CODE REVIEW

File: queries/php/highlights.scm [hunk 1]
```diff
@@ -108,6 +108,12 @@
 
 ; Conditions ( ? : )
 (conditional_expression) @conditional
+
+; Directives
+(declare_directive "strict_types" @preproc)
+(declare_directive "ticks" @preproc)
+(declare_directive "encoding" @preproc)
```
Comment (lucario387) [inline queries/php/highlights.scm:115]:
```suggestion
(declare_directive ["strict_types" "ticks" "encoding"] @preproc)
```

Comment (lucario387) [reply queries/php/highlights.scm:115]: And are there other keywords for declare?
If there is then maybe you should do something else

Comment (zweimach) [reply queries/php/highlights.scm:115]: I didn't know this syntax, thanks!

Comment (zweimach) [reply queries/php/highlights.scm:115]: There are only these three directives in [the parser](https://github.com/tree-sitter/tree-sitter-php/blob/973694ffcdeebca245b7ecf0d7c4cadd4f41b3c9/grammar.js#L566).

**Comment by gbprod** [comment]
IMHO, `@parameter` would be more suitable (like for named arguments) as it's not really a preprocessor directive.
What do you think ?

**Comment by zweimach** [comment]
> IMHO, `@parameter` would be more suitable (like for named arguments) as it's not really a preprocessor directive.

Yes, it could be.

I picked `@preproc` because IMO directives are similar to `#pragma` in C/C++.
What do you think @tk-shirasaka?

**Comment by tk-shirasaka** [comment]
I agree with `@parameter`.

I think that if the syntax is expanded, it is easy to be interpreted differently by each person.
(I also thought `ini_set` was semantically close).

**Review by theHamsta — APPROVED** [review]

**Pull request merged**